index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset=utf8>
+  NIN_WILL_REPLACE_THIS_TAG_WITH_AUTOGENERATED_COMMENT_TAGS
```

Comment:
This should be kept like it stands. nin will automatically replace it with values from nin.json on compile time.